content/posts/macvdmtool.md

```diff
@@ -57,8 +61,10 @@ It is a tool written by [marcan](https://github.com/marcan) under the [AsahiLinu
     Entering DBMa mode... Status: DBMa
     Rebooting target into DFU mode... OK
     Exiting DBMa mode... OK
-    ~ %
     ```
+
+## Restoring/reviving
+As well as using Apple Configurator 2 or in [latest Sonoma update, Finder](https://www.macrumors.com/2023/08/15/macos-sonoma-dfu-mode/) to restore/revive your Mac, you can also do it manually with cfgutil (only works if the target is an Apple Silicon model)
```

Comment:
```suggestion
> Alternatively you can use Apple Configurator 2 or [Finder](https://www.macrumors.com/2023/08/15/macos-sonoma-dfu-mode/) to restore/revive your Mac. 

> **Note**: This method with `cfgutil` only works if the target Mac is Apple Silicon.
```

Reply:
I just used macvdmtool again today for the first time in like ~1 year and... wow the new Finder stuff for restoring is awesome. I'm thinking to update the blog post to include it. Or to re-write the blog post to skip all the CLI stuff.